lib/krikri/index_querier.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri
+  # Queries the Search Index
+  class IndexQuerier
```

Comment:
@anarchvist I created this class because I found that I kept repeating these three lines of code:

<pre>@repo = Blacklight::SolrRepository.new(Blacklight::Configuration.new
query_result = @repo.search(query_params)
Blacklight::SolrResponse.new(query_result, query_params)</pre>

We have three libraries so far that need to query the Solr index, and I believe there will be more.  Since we will eventually be moving to ElasticSearch, I am trying to keep the logic that deals with the SolrIndex as isolated as possible.  Maybe a class isn't best way to handle this.  I could just add this logic back into the individual libraries that use it, or I could make a module instead of a class.  I defer to your good judgement, and am happy to discuss further if it's not yet clear.